src/components/startworkout/WeightAndRapField.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const WeightAndRapField = ({ setDetails }: { setDetails: string }) => {
```

Comment:
```suggestion
const RecordInput = ({ field }: { field: string }) => {
```
コンポーネント名はこんな感じの方がシンプルかなと思いますがどうですか？

あと`set~`みたいな名前はstateセット関数と似ていて混乱しそうなので、何のインプットフィールドなのかを引数で渡す感じだとシンプルで分かりやすいかと思います

Reply:
propsに渡される値は決まっているのでtype指定してもいいかもです
 ```
type RecordInputProps = {
  field: "weight" | "rep"
}
```